### PR TITLE
Fix: prevent permanent loading overlay when switching languages (fixes #7)

### LINKED ISSUE #7: When switching languages a permanent loading overlay is displayed
### Subject of the issue
When switching languages (with [adapt-contrib-languagePicker](https://github.com/adaptlearning/adapt-contrib-languagePicker/)) a permanent loading overlay is displayed.

### Your environment
* all framework versions, browser and devices

### Steps to reproduce
Add this plugin to a multilingual course and enable it, then attempt to switch languages.

### PR DESCRIPTION
**Pull request by chris-steele**
#7 

### Fix
* prevent permanent loading overlay when switching languages

### CODE REVIEW

**Comment by oliverfoster** [comment]
setupOverrides is also executing twice. That needs some consideration.

**Comment by joe-allen-89** [comment]
@chris-steele can you take a look at the above comment around setupOverrides executing twice. Thanks.

**Comment by chris-steele** [comment]
@oliverfoster @joe-allen-89 amended as requested - this will now account for the sitation where one language uses transitions and another does not.

**Comment by oliverfoster** [comment]
You can junk off the ie11 stuff now. It's not necessary any more.

File: js/adapt-contrib-contentObjectTransition.js [hunk 1]
```diff
@@ -175,7 +143,6 @@ class ContentObjectTransition extends Backbone.Controller {
     incomingContentObject.$el[0].removeEventListener('transitionend', this.endAnimation);
     incomingContentObject.$el[0].removeEventListener('animationend', this.endAnimation);
 
-    // Use raf to stop any perceptible jumping in IE11
     window.requestAnimationFrame(() => {
```
Comment (cahirodoherty-learningpool) [inline js/adapt-contrib-contentObjectTransition.js:146]: Can this whole piece go if it was IE11 specific?

Comment (chris-steele) [reply js/adapt-contrib-contentObjectTransition.js:146]: I think the block is needed - without it the animation won't work correctly. Might be that `requestAnimationFrame` was the implementation choice to suit IE11, but it still works for modern browsers. @oliverfoster any comment on this?

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by joe-allen-89 — CHANGES_REQUESTED** [review]
As we're removing IE11 could you also remove it from the readme and bump the framework version to v5.20.0.

Whilst your editing the readme would you also be so kind as to remove the plugin version to be consistent with other plugins, thanks.

**Review by joe-allen-89 — APPROVED** [review]
👍

**Review by lemmyadams — APPROVED** [review]

**Pull request merged**